ui/app/templates/components/calendar-widget.hbs

```diff
@@ -58,34 +59,29 @@
     {{#if this.showCalendar}}
       <div class="calendar-widget-container" data-test-calendar-widget-container>
         <div class="select-year">
-          <button
+          <Hds::Button
```

Comment:
Year arrows:
<img width="1440" alt="client-count-compare" src="https://github.com/hashicorp/vault/assets/82459713/f5214315-da53-404d-9ebb-9157118f021d">

Reply:
Did you decide against the size change?